pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -633,6 +633,8 @@ async def run_repomdrecord_download(name, location_href, downloader):
                         metadata_pb.increment()
                 except ClientResponseError as exc:
                     raise HTTPNotFound(reason=_("File not found: {}".format(exc.request_info.url)))
+                except FileNotFoundError:
```

Comment:
I might be missing the context, but in my personal workaround, I didn't need this bit to fix the issue.  Only the two 
```python
except (ClientResponseError, FileNotFoundError):
``` 
parts.

Reply:
This isn't related to the fix, it's just being explicit about the errors that could be thrown here and what we're doing with them.